apps/judicial-system/backend/src/app/modules/court/court.service.ts

```diff
@@ -117,12 +129,15 @@ export class CourtService {
     fileName: string,
     content: Buffer,
   ): Promise<string> {
-    return this.uploadStream(
-      courtId,
-      `${fileName}.pdf`,
-      'application/pdf',
-      content,
-    )
+    return this.confirmApiAvailability()
```

Comment:
I feel like there should be a safer way of doing this. One could easily forget to call this method first before trying to access `this.courtClientService`. When this config is set, could we inject a striped-out implementation of the `CourtClientService`?

Reply:
You are right of course, but I am not sure it is worth the effort. I use the same approach in the police service when calling LÖKE. I'm going to ponder this a bit and see if I can easily come up with a more elegant solution.

Reply:
I see, I will take the auto merge label off and accept. Then you can let this through if you want.